src/components/FindMovie/FindMovie.tsx

```diff
@@ -1,10 +1,70 @@
-import React from 'react';
+import classNames from 'classnames';
+import React, { ChangeEventHandler, FormEventHandler, useState } from 'react';
+import { Movie } from '../../types/Movie';
+import { getMovie } from '../../api';
+import { MovieData } from '../../types/MovieData';
+import { ResponseError } from '../../types/ReponseError';
 import './FindMovie.scss';
+import { MovieCard } from '../MovieCard';
+
+// eslint-disable-next-line max-len
+const DEFAULT_PREVIEW = 'https://via.placeholder.com/360x270.png?text=no%20preview';
```

Comment:
just a suggestion
usually such things are placed to some neighboring file, like `FindMovie.constants.ts`